packages/core/src/icons/icons.stories.js

```diff
@@ -2,20 +2,22 @@ import path from 'path';
 
 import './all.css';
 import './icon.css';
+import { storiesOf } from "@storybook/html";
```

Comment:
Suggestion: should this be before all the other imports?

Reply:
- [ ] Move import to before other imports